master/internal/api_auth.go

```diff
@@ -174,28 +174,27 @@ func processProxyAuthentication(c echo.Context) (done bool, err error) {
 	return err != nil, authz.SubIfUnauthorized(err, serviceNotFoundErr)
 }
 
-// processAuthWithRedirect is an auth middleware that redirects the requests
+// processAuthWithRedirect is an auth middleware that redirects browser requests
 // to login page for a set of given paths in case of authentication errors.
 func processAuthWithRedirect(redirectPaths []string) echo.MiddlewareFunc {
 	return func(next echo.HandlerFunc) echo.HandlerFunc {
 		return func(c echo.Context) error {
```

Comment:
nit: it's outside of the scope of this PR ... but it's usually more idiomatic to use `ctx` for context and `c` for channel in Golang. It looks like this file uses a mix of `c` and `ctx`, probably because some functions have multiple contexts? In that case, we can name a context more specifically, e.g. `reqCtx`. 

I'm not suggesting it needs to be changed now, but maybe something to keep in mind for future code changes for readability?

Reply:
hmm I never put too much thought into this particular case but from a quick search now it looks like in our code base we almost always use `c` for `echo.Context` but never for `context.Context` 🤷🏼 
if we do migrate a find and replace should be enough to migrate all of these.
```
hbp ➜ determined git:(hz-proxied-err-handling) ag 'ctx echo.context' | wc -l
       3
hbp ➜ determined git:(hz-proxied-err-handling) ag 'c echo.context' | wc -l
      54
```

Reply:
Oh, that makes sense; sounds like it's a convention. 👍 Thanks for looking into it!